alpaca/data/historical/stock.py

```diff
@@ -72,6 +73,25 @@ def __init__(
             raw_data=raw_data,
         )
 
+    def get_stock_auctions(self, request_params: StockAuctionsRequest):
+        """Returns auction data for an equity or list of equities over a given time period.
+
+        Args:
+            request_params (StockAuctionsRequest): The request object for retrieving stock auction data.
+
+        Returns:
+            Union[AuctionSet, RawData]: The auction data either in raw or wrapped form
+        """
+        raw_auctions = self._get_marketdata(
+            path="/stocks/auctions",
+            params=request_params.to_request_fields(),
+        )
```

Comment:
We have recently introduced `page_size` parameter to `_get_marketdata()` [1], could you please set page_size for auctions as well?

*1 https://github.com/alpacahq/alpaca-py/pull/583
```suggestion
        raw_auctions = self._get_marketdata(
            path="/stocks/auctions",
            params=request_params.to_request_fields(),
            page_limit = 1_000,
            page_size = 1_000,
        )
```